src/app/App.js

```diff
@@ -20,6 +20,13 @@ import {
 import { useScroll } from './controllers';
 import { Footer, Nav } from './components';
 
+import "./styles/global.scss"
+import "./styles/colors.scss"
+import "./styles/layout.scss"
+import "./styles/components.scss"
+import "./styles/animations.scss"
+import "./styles/forms.scss"
```

Comment:
could pref. having a main.scss file that does all of these imports for us like in the section that talks about main sass files here: https://sass-guidelin.es/#the-7-1-pattern

but honestly don't mind either way.

Reply:
agreed, I might bundle these together eventually, but I am also working on reorganizing styles to be closer to their associated components, so some of these files will be removed